src/styles/_Carousel.scss

```diff
@@ -25,9 +34,10 @@
 }
 
 .game-card {
```

Comment:
When I search for a title, the fewer titles there are the taller the cards become. Not sure what direction to take to fix. Maybe a max-height combined with % height? height 280px max-height:10%?

Reply:
Yes you're right! I didn't catch that. Tomorrow lets try either all fixed heights or all percentages.